<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ListWindow</class>
 <widget class="QMainWindow" name="ListWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1300</width>
    <height>671</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Liste des images</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTableWidget" name="databaseTable">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>741</width>
      <height>221</height>
     </rect>
    </property>
    <column>
     <property name="text">
      <string>Nom de l'image</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Source</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Auteur</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Coût</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Niveau d'accès</string>
     </property>
    </column>
   </widget>
   <widget class="QPushButton" name="exitButton">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>360</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Exit</string>
    </property>
   </widget>
   <widget class="QPushButton" name="addImageButton">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>360</y>
      <width>221</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Ajouter une image</string>
    </property>
   </widget>
   <widget class="QPushButton" name="imageProcessingButton">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>360</y>
      <width>241</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Effectuer un traitement </string>
    </property>
   </widget>
   <widget class="QGroupBox" name="imageDisplayBox">
    <property name="geometry">
     <rect>
      <x>850</x>
      <y>10</y>
      <width>261</width>
      <height>231</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1300</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
